atlasdb-client/src/main/java/com/palantir/atlasdb/cache/OffHeapTimestampCache.java

```diff
@@ -154,47 +156,45 @@ private static MetricName constructCacheMetricName(String metricSuffix) {
                 .build();
     }
 
-    private static class WriteBatcher
-            implements CoalescingRequestFunction<Map.Entry<Long, Long>, Map.Entry<Long, Long>> {
+    private static class WriteBatcher implements CoalescingRequestFunction<Map.Entry<Long, Long>, Void> {
         OffHeapTimestampCache offHeapTimestampCache;
 
         WriteBatcher(OffHeapTimestampCache offHeapTimestampCache) {
             this.offHeapTimestampCache = offHeapTimestampCache;
         }
 
         @Override
-        public Map<Map.Entry<Long, Long>, Map.Entry<Long, Long>> apply(Set<Map.Entry<Long, Long>> request) {
+        public Map<Map.Entry<Long, Long>, Void> apply(Set<Map.Entry<Long, Long>> request) {
             CacheDescriptor cacheDescriptor = offHeapTimestampCache.cacheDescriptor.get();
             if (cacheDescriptor.currentSize().get() >= offHeapTimestampCache.maxSize.getAsLong()) {
                 offHeapTimestampCache.taggedMetricRegistry.counter(CACHE_NUKE).inc();
                 offHeapTimestampCache.clear();
             }
             cacheDescriptor = offHeapTimestampCache.cacheDescriptor.get();
             try {
+                List<Long> toWrite = request.stream().map(Map.Entry::getKey).collect(Collectors.toList());
                 Map<Long, Long> response = offHeapTimestampCache.timestampStore.get(
-                        cacheDescriptor.storeNamespace(),
-                        request.stream().map(Map.Entry::getKey).collect(Collectors.toList()));
-
-                Map<Long, Long> toWrite = ImmutableMap.copyOf(Sets.difference(request, response.entrySet()));
-                offHeapTimestampCache.timestampStore.put(
-                        cacheDescriptor.storeNamespace(),
+                        cacheDescriptor.entryFamily(),
                         toWrite);
 
-                cacheDescriptor.currentSize().addAndGet(toWrite.size());
+                int sizeIncrease = Sets.difference(request, response.entrySet()).size();
+                offHeapTimestampCache.timestampStore.put(cacheDescriptor.entryFamily(), ImmutableMap.copyOf(request));
```

Comment:
nit: move this next to `cacheDescriptor.currentSize...`

Reply:
I think I moved it to correct position. Please double check.